gomint-api/src/main/java/io/gomint/entity/EntityPlayer.java

```diff
@@ -319,4 +319,30 @@
      */
     boolean getFlying();
 
+    /**
+     *
+     * @param title
+     * @param subtitle
+     * @param fadein
+     * @param duration
+     * @param fadeout
+     */
+
+    void sendTitle( String title, String subtitle, int fadein, int duration, int fadeout );
```

Comment:
All gomint-api is not based on ticks. Please use long, TimeUnit when you need duration times (in this case for fadeIn, duration and fadeOut)